intel-retail/retail-use-cases
# feat: Add Pytorch yolov5 object detection pipelines

## PR Checklist



## What are you changing?
Added Pytorch yolov5 object detection pipeline with performance evaluation feature

## Issue this PR will close

close: #**46**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- Make sure input source video is added to performance-tools/sample-media folder and update path in https://github.com/NeethuES-intel/retail-use-cases/blob/f48213aa16c5d96fcb56a415c3120780e00ce804/use-cases/demos/add_demo_pytorch_object_detection.yml#L21
- make build-pytorch
- make run_pytorch_object_detection
- run 2 pipelines with PIPELINE_COUNT=2 make run_pytorch_object_detection
- make down_pytorch_object_detection
- results of the detection (only object detected and number for now) is also published to mqtt broker (topic - pytorch_yolov5_results) and logged into results.txt 

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/retail-use-cases )
